source/SH3/system/glprogram.cpp

```diff
@@ -48,6 +48,44 @@ std::string sh3_glprogram::load_error::message() const
     return error;
 }
 
+
+void sh3_glprogram::Load(const std::string& name)
```

Comment:
Is this method not equivalent to the other Load-overload when called with an empty `attribs` vector?